From: Oswald Buddenhagen <ossi@kde.org>
Date: Sat, 27 Oct 2012 07:58:16 +0000
Subject: remove unnecessary if() statement
X-Git-Url: http://quickgit.kde.org/?p=phonon.git&a=commitdiff&h=bbbec9961a3e1bfd68b98c0521b38c5541079522
---
remove unnecessary if() statement

as it happens, this works around a qmake breakage relating to parsing if()

BUG: 295037
FIXED-IN: 4.6.1
---


--- a/qt_phonon.pri
+++ b/qt_phonon.pri
@@ -3,7 +3,7 @@
 # the next build of Qt. So pretend we're not there in this case.
 # This file is loaded by qt_config.prf, before .qmake.cache has been loaded.
 # Consequently, we have to do some stunts to get values out of the cache.
-if(!exists($$_QMAKE_CACHE_)| \
-   !contains($$list($$fromfile($$_QMAKE_CACHE_, CONFIG)), QTDIR_build)): \
+!exists($$_QMAKE_CACHE_)| \
+   !contains($$list($$fromfile($$_QMAKE_CACHE_, CONFIG)), QTDIR_build): \
     QT_CONFIG += phonon
 

